### PR TITLE
 #79086 Appointment Details Redesign - Content / CTA updates (view appointment details)

### PR DESCRIPTION
**Pull request by vbahinwillit**
## Summary
This PR updates the content and layout for the appointment details page.

## Related issue(s)

- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#79086

## Testing done
Unit and e2e testing

## Screenshots
**NOTE**: There are too many screenshots to add.

localhost_3001_my-health_appointments_canceled-comp-appointment(iPhone SE) (2)
---
localhost_3001_my-health_appointments_canceled-comp-appointment(iPhone SE) (3)
---
localhost_3001_my-health_appointments_canceled-comp-appointment(iPhone SE) (4)

## What areas of the site does it impact?
VAOS

## Acceptance criteria
All tests must pass

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by ryanshaw — DISMISSED** [review]
🥳!

**Comment by vbahinwillit** [comment]
Thank you Sir!!!

File: src/applications/vaos/components/StatusAlert.jsx [hunk 1]
```diff
@@ -43,62 +44,139 @@ export default function StatusAlert({ appointment, facility }) {
...
-            headline="We can't access after-visit summaries at this time."
-          >
-            We’re sorry. We’ve run into a problem.
-          </InfoAlert>
+        <InfoAlert backgroundOnly status="info">
+          <p>
+            We’ll try to schedule your appointment in the next 2 business days.
+            Check back here or call your facility for updates. Review your
+            appointments Schedule a new appointment
+          </p>
```
Comment (jenniemc) [inline src/applications/vaos/components/StatusAlert.jsx:67]: The last sentence "Review your appointments Schedule a new appointment" can be removed.

File: src/applications/vaos/components/StatusAlert.jsx [hunk 2]
```diff
@@ -43,62 +44,139 @@ export default function StatusAlert({ appointment, facility }) {
...
+        </InfoAlert>
+      );
+    }
+    return (
+      <InfoAlert backgroundOnly status="success">
+        <p>
+          We’ll try to schedule your appointment in the next 2 business days.
+          Check back here or call your facility for updates. Review your
+          appointments Schedule a new appointment
+        </p>
```
Comment (jenniemc) [inline src/applications/vaos/components/StatusAlert.jsx:77]: Same as above. The last sentence "Review your appointments Schedule a new appointment" can be removed.

File: src/applications/vaos/components/StatusAlert.jsx [hunk 3]
```diff
@@ -43,62 +44,139 @@ export default function StatusAlert({ appointment, facility }) {
...
+
+    if (APPOINTMENT_STATUS.proposed === status) {
+      if (isPastAppointment) {
+        return (
+          <InfoAlert backgroundOnly status="info">
+            <p>
+              We’ll try to schedule your appointment in the next 2 business
+              days. Check back here or call your facility for updates. Review
+              your appointments Schedule a new appointment
+            </p>
```
Comment (jenniemc) [inline src/applications/vaos/components/StatusAlert.jsx:121]: Same as above. The last sentence "Review your appointments Schedule a new appointment" can be removed.

File: src/applications/vaos/components/StatusAlert.jsx [hunk 4]
```diff
@@ -43,62 +44,139 @@ export default function StatusAlert({ appointment, facility }) {
...
       return (
-        <p className="vads-u-margin--0">
-          An after-visit summary is not available at this time.
-        </p>
+        <InfoAlert backgroundOnly status="success">
+          <p>
+            We’ll try to schedule your appointment in the next 2 business days.
+            Check back here or call your facility for updates. Review your
+            appointments Schedule a new appointment
+          </p>
```
Comment (jenniemc) [inline src/applications/vaos/components/StatusAlert.jsx:131]: Same as above. The last sentence "Review your appointments Schedule a new appointment" can be removed.

**Review by jenniemc — DISMISSED** [review]
@vbahinwillit, Looks terrific!! Have a some minor text removal your next iteration.

**Comment by vbahinwillit** [comment]
Thanks Jenny!!! Appreciate it as always!

File: src/applications/vaos/components/StatusAlert.jsx (hunk 3 above)
Comment (vbahinwillit) [reply src/applications/vaos/components/StatusAlert.jsx:121]: Done

File: src/applications/vaos/components/StatusAlert.jsx (hunk 4 above)
Comment (vbahinwillit) [reply src/applications/vaos/components/StatusAlert.jsx:131]: Done

File: src/applications/vaos/components/StatusAlert.jsx (hunk 2 above)
Comment (vbahinwillit) [reply src/applications/vaos/components/StatusAlert.jsx:77]: Done

File: src/applications/vaos/components/StatusAlert.jsx (hunk 1 above)
Comment (vbahinwillit) [reply src/applications/vaos/components/StatusAlert.jsx:67]: Done.

File: src/applications/vaos/components/tests/PhoneLayout.unit.spec.js [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+      expect(screen.getByText(/Facility details not available/i));
+      expect(screen.container.querySelector('va-icon[icon="directions"]')).not
+        .to.exist;
+      expect(
+        screen.getByRole('link', {
+          name: /Find facility information Link opens in a new tab/i,
+        }),
+      );
+
+      // TODO: Check with Ciera on this.
```
Comment (JunTaoLuo) [inline src/applications/vaos/components/tests/PhoneLayout.unit.spec.js:164]: Are you planning to resolve this before merging this PR?

**Review by JunTaoLuo — APPROVED** [review]
I love seeing all the new tests. Great work! Signing off for the test portions on this PR but someone more familiar with the other components should also take a final look.

File: src/applications/vaos/components/tests/PhoneLayout.unit.spec.js (hunk 5 above)
Comment (vbahinwillit) [reply src/applications/vaos/components/tests/PhoneLayout.unit.spec.js:164]: @JunTaoLuo - Done

**Pull request merged**